dev/devicelab/README.md

```diff
@@ -211,31 +195,30 @@ A task runs in its own standalone Dart VM and reports results via Dart VM
 service protocol. This ensures that tasks do not interfere with each other and
 lets the CI system time out and clean up tasks that get stuck.
 
-## Adding tests to the CI environment
+## Adding tests to continuous integration
 
-The `manifest.yaml` file describes a subset of tests we run in the CI. To add
-your test edit `manifest.yaml` and add the following in the "tasks" dictionary:
-
-```
-  {NAME_OF_TEST}:
-    description: {DESCRIPTION}
-    stage: {STAGE}
-    required_agent_capabilities: {CAPABILITIES}
-```
+Host only tests should be added to `flutter_tools`.
```

Comment:
It sounds like the following instructions are to add it to `flutter_tools` but the rest of this section is how to not add it to flutter_tools...

How about removing this sentence and add at the bottom of the section something like:
"Note that DeviceLab tests should generally require a tethered device.  If you are adding host-only tests, considering adding your test to `packages/flutter_tools/test/integration.shard` ."

Reply:
Thanks, updated.